configure.ac

```diff
@@ -2064,6 +2064,9 @@ fi
 if test "$use_alsa" = "yes"; then
   USE_ALSA=1
   AC_DEFINE([USE_ALSA],[1],["Define to 1 if alsa is installed"])
+  if test "$ac_cv_func_eventfd_read" = "yes" -a "$ac_cv_func_eventfd_write" = "yes"; then
```

Comment:
I am not a autotools guru but think something like AC_CHECK_FUNCS should do for automatic detection of availability